storage-bigtable/src/access_token.rs

```diff
@@ -105,13 +109,20 @@ impl AccessToken {
                 .compare_and_swap(false, true, Ordering::Relaxed)
             {
                 // Refresh already pending
+                let token_refresh_time = self.token_refresh_start_time.load(Ordering::SeqCst);
+                if token_refresh_time != 0 && token_refresh_time + (self.get_token_timeout_seconds * 2) < token_r.1.elapsed().as_secs() {
+                    warn!("Token refresh timeout failed to timeout!");
```

Comment:
This message is confusing. I would suggest remote "to timeout" or make it more descriptive.